docker/entrypoint_check.sh

```diff
@@ -1,18 +1,31 @@
 #!/bin/sh -l
 
 echo '\nGetting the code...\n'
-git clone --quiet https://$INPUT_TOKEN@github.com/$GITHUB_REPOSITORY check
+git clone --quiet https://$PAT@github.com/$GITHUB_REPOSITORY check
 cd check
-git checkout $BRANCH_SOURCE
+git checkout $GITHUB_HEAD_REF
 rm .Rprofile
 
 echo '\nSession info\n'
 Rscript -e 'sessioninfo::session_info()'
 
+echo '\nCheck if only protocol is updated...\n'
+FOLDERNAME=$(Rscript -e 'protocolhelper::get_path_to_protocol("'$GITHUB_HEAD_REF'")')
+FOLDERNAME_REL=${FOLDERNAME#/home/runner/work/protocolsource/}
+echo 'FOLDERNAME=' $FOLDERNAME
+echo 'FOLDERNAME_REL=' $FOLDERNAME_REL
+CHANGED=$(git diff --name-only main | grep -v NEWS\.md | grep -v \.zenodo\.json| grep -v ^$FOLDERNAME_REL/)
+echo 'CHANGED=' $CHANGED
+
+if [ -n "${CHANGED}" ]; then
+  echo '\nFiles not belonging to the protocol mentioned in the branch name are changed, and these changes will not be passed to the website. Please remove these changes here and start a new branch for changes to other protocols./n\n'
```

Comment:
```suggestion
  echo '\nFiles not belonging to the protocol mentioned in the branch name ("'$GITHUB_HEAD_REF'") are changed, and these changes will not be passed to the website. Please remove these changes from this branch and start a new branch for changes to other protocols./n\n'
```

Reply:
Does `$CHANGED` gives a list of which files were changed? If so, maybe include it at the end of this message?

Reply:
With respect to the error: would it be a good idea to just replace the whole description `to the protocol mentioned in the branch name` by the branch name?  In case the user has the right protocol code as a branch name, it would be easier to understand.  On the other hand, when he made a mistake in the branch name, only mentioning the branch name and not the description, would make it difficult to understand why the error message is dropped...

Reply:
> Does `$CHANGED` gives a list of which files were changed? If so, maybe include it at the end of this message?

Yes it does.  I'll add it, do you give your opinion on my previous comment?

Reply:
> With respect to the error: would it be a good idea to just replace the whole description `to the protocol mentioned in the branch name` by the branch name? In case the user has the right protocol code as a branch name, it would be easier to understand. On the other hand, when he made a mistake in the branch name, only mentioning the branch name and not the description, would make it difficult to understand why the error message is dropped...

Maybe reword it as follows:

'\nBranch '$GITHUB_HEAD_REF' contains changes to files which do not belong to protocol '$GITHUB_HEAD_REF', and these changes will not be passed to the website. Please remove these changes from this branch and start a new branch for changes to other protocols./n\n'

Reply:
Ok, and do I keep the `changes to be removed: xxx` in the end?